CMakeLists.txt

```diff
@@ -35,11 +35,6 @@ set(ROCMLIR_BIN_DIR "${CMAKE_CURRENT_BINARY_DIR}/bin" CACHE PATH "")
 set(ROCMLIR_LIB_DIR "${CMAKE_CURRENT_BINARY_DIR}/lib" CACHE PATH "")
 message(STATUS "ROCMLIR_BIN_DIR: ${ROCMLIR_BIN_DIR}")
 
-if(BUILD_MIXR_TARGET)
```

Comment:
We might still want to support BUILD_MIXR_TARGET as a flag for backwards-compatibility reasons